src/main/pages/che-6/user-guide/version-control.adoc

```diff
@@ -9,55 +9,85 @@ folder: che-6/user-guide
 
-*Note: HTTPS git URL can only be used to push changes if you enable oAuth authentication as described in link:user-management.html#social-login-and-brokering[GitHub oAuth]*.
+[NOTE]
+====
+HTTPS Git URL can only be used to push changes if you enable oAuth authentication as described in link:user-management.html#social-login-and-brokering[GitHub oAuth].
+====
 
-[id="generateupload-ssh-keys"]
-== Generate/Upload SSH Keys
+[id="generating-and-uploading-ssh-keys"]
+== Generating and uploading SSH keys
 
-SSH keys can be generated at `Profile > Preferences > SSH > VCS`. Use the `Generate Key` button and manually save the resulting key to your Git hosting provider account. When prompted to provide the hostname for your repo, make sure it is a bare hostname (no www or http/https) as in the example below. After the key has been generated, you can view and copy it, and save to your repository hosting account.
+To generate the SSH keys, take the following steps:
 
-*IMPORTANT!*. Hostname is an actual hostname of a VCS provider. Examples: `github.com`, `bitbucket.org`.
+. Click *Profile* > *Preferences* > *SSH* > *VCS*.
 
-*Use Existing SSH Keys*
+. Click *Generate Key* and manually save the resulting key to your Git hosting provider account. 
 
-You can upload an existing public key instead of creating a new SSH key. When uploading a key add the hostname (using no *www or http/https* - as in the example below). Note that the `public key > view` button will not be available with this option as the public file should be generated already.
+. When prompted to provide the hostname for your repository, ensure that it is a bare hostname (without www or http/https). Hostname is an actual hostname of a VCS provider. Example: `github.com`, `bitbucket.org`
+
+After the key is generated, you can view and copy it, and save it to your repository hosting account.
+
+[id='using-existing-ssh-keys']
+=== Using existing SSH keys
+
+You can upload an existing public key instead of creating a new SSH key. When uploading a key, add the hostname (without www or http/https). Note that the *View* button in the *Public Key* column will not be available with this option because the public file should be generated already.
 
 image::git/ssh_keys.gif[]
 
-The following example is specific to GitHub and GitLab but can be used with all git or SVN repository providers that use SSH authentication. Please refer to documentation provided by other providers for additional assistance.
+The following example is specific to GitHub and GitLab, but it can be used with any Git or SVN repository providers that use SSH authentication. For additional information, refer to the documentation provided by other providers.
+
+*GitHub example*
+
+If you do not want to set up GitHub oAuth, you can manually upload a key. Add the associated public key to a repository or account using *github.com*. 
```

Comment:
Here, `github.com` is marked up with bold styling -- we should be consistent with this and use one style or another, not both.